_docs/installation.md

```diff
@@ -23,6 +23,12 @@ Pick a set of instructions based on your operating system.
 ~ $ mkdir -p ~/.clusterrunner/dist && cd ~/.clusterrunner && curl -L https://cloud.box.com/shared/static/2pl4pi6ykvrbb9d06t4m.tgz > clusterrunner.tgz && tar -zxvf clusterrunner.tgz -C ./dist && cp ./dist/conf/default_clusterrunner.conf clusterrunner.conf && chmod 600 clusterrunner.conf
 {% endhighlight %}
 
+**Windows**
+
+- Download ClusterRunner from [here](https://cloud.box.com/shared/static/u91zg1fmmlxo2reqo8mwi3hjv7kaeazz.zip)
```

Comment:
I plan to make the installation easier like Chocolatey (https://chocolatey.org/) later.